.github/workflows/pgcmp.yml

```diff
@@ -42,6 +42,7 @@ jobs:
         run: sudo curl -L https://github.com/hasura/graphql-engine/raw/stable/cli/get.sh | bash
       - name: Start Postgres
         run: |
+          sed -i 's/postgres:14.8/postgres:16.2/' docker-compose.yml
```

Comment:
I wonder, should we have migrations just start from `v2.9.0` since the only thing we changed between that and `v2.8.0` was upgrading to PG 16.2?

Reply:
We shouldn't, as 2.9.0 released with migration 1. 2.8.0 is the true "base" of the DB.

Reply:
Ah right, looks like our `2.9.0` release changelog is a little funky, it's showing the diff from `2.6.103`